### PR TITLE
comment edits of seqid3, seqid, seqid2, df-gsum

### PR DESCRIPTION
**Pull request by benjub**
* comment edits of seqid3, seqid, seqid2, df-gsum
* rewraps of some nearby statements (when the math-expression after the typecode '|-' fits on a single line, then write it on a single line)

(this is the first half of the recently closed #2514)

(edit: should be merged before I push the second half, since I changed the capitalization of one section title and the second half will modify indentation of section titles)

### CODE REVIEW

File: set.mm [hunk 1]
```diff
@@ -149685,11 +149685,11 @@ seq M ( .+ , F ) ) $=
     seradd.2 $e |- ( ( ph /\ k e. ( M ... N ) ) -> ( F ` k ) e. CC ) $.
     seradd.3 $e |- ( ( ph /\ k e. ( M ... N ) ) -> ( G ` k ) e. CC ) $.
     seradd.4 $e |- ( ( ph /\ k e. ( M ... N ) ) ->
-                     ( H ` k ) = ( ( F ` k ) + ( G ` k ) ) ) $.
+                                     ( H ` k ) = ( ( F ` k ) + ( G ` k ) ) ) $.
     $( The sum of two infinite series.  (Contributed by NM, 17-Mar-2005.)
```
Comment (avekens) [inline set.mm]: I think there is no convention/agreement that a (follow up) line should always be right aligned. I would prefer the following line break/indentation (the dots in the second lien are only for displaying purposes in this comment):
```metamath
seradd.4 $e |- ( ( ph /\ k e. ( M ... N ) )
................................-> ( H ` k ) = ( ( F ` k ) + ( G ` k ) ) ) $.
```

**Review by avekens — APPROVED** [review]

File: set.mm (hunk 1 above)
Comment (benjub) [reply set.mm]: You're right that there is no agreement on that point, and I might have abused the editing towards my preference.  I'll refrain from doing it.

Comment (digama0) [reply set.mm]: I prefer the original indentation (which I think is also mmj2 style). Right alignment is way too much work to keep in sync.

Comment (benjub) [reply set.mm]: Ok. Reverted these alignments (the ones where the mmj2-style indentation was followed)

**Review by tirix — APPROVED** [review]

**Pull request merged**